test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -166,21 +166,42 @@ def test_pip_check(image):
 
     :param image: ECR image URI
     """
-    # Add null entrypoint to ensure command exits immediately
     ctx = Context()
     gpu_suffix = '-gpu' if 'gpu' in image else ''
 
     # TF inference containers do not have core tensorflow installed by design. Allowing for this pip check error
     # to occur in order to catch other pip check issues that may be associated with TF inference
     allowed_exception = re.compile(rf'^tensorflow-serving-api{gpu_suffix} \d\.\d+\.\d+ requires '
                                    rf'tensorflow{gpu_suffix}, which is not installed.$')
+
+    # Add null entrypoint to ensure command exits immediately
     output = ctx.run(f"docker run --entrypoint='' {image} pip check", hide=True, warn=True)
     if output.return_code != 0:
         if not allowed_exception.match(output.stdout):
             # Rerun pip check test if this is an unexpected failure
             ctx.run(f"docker run --entrypoint='' {image} pip check", hide=True)
 
 
+@pytest.mark.model('N/A')
```

Comment:
nit: add `pytest.mark.integration('pandas')`

Reply:
+1, or `package-sanity-test`

Reply:
just kept as pandas to be specific -- directory structure shows it's a sanity test